### ISSUE TITLE
NTR: Abdominal imaging: peri-hepatic enhancement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9043]
New term request Abdominal imaging: peri-hepatic enhancement (nadea3_210827112408): Perihepatitis; abdomen

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000148
- label: Perihepatic enhancement
- parents: [Abnormal liver morphology (HP:0410042)](https://hpo.jax.org/app/browse/term/HP:0410042)
- pmid: PMID:35528764
- def: Increased signal (contrast enhancement) surrounding the liver observed on computer tomography or magnetic resonance imaging following administration of contrast agents.
- comment: See figure 2 of PMID:35528764.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9043
- synonyms: Peri-hepatic enhancement(NONE)
